config/statement/assessment_export_options.yml

```diff
@@ -60,6 +60,8 @@ defaults:
 original_statements:
   docx:  false
   pdf:
+    _defaults:
```

Comment:
Without these changes, the defaults we get in the FE look like this:

```
anonymous:true
exportType:"statementsOnly"
sortType:"false"
template:"portrait"
```

(i.e., the defaults stored under `defaults.pdf._defaults` are applied)

With these changes, it's only 

`newPagePerStn: "false"`

Since this is the core yml, I think we should include the previous options as well.

Reply:
adjusted in [4da07c4](https://github.com/demos-europe/demosplan-core/pull/3578/commits/4da07c4571dbea91c6dc4c3808596d31bb222614)